### PR TITLE
Added new design for ND explore bar

### LINKED ISSUE #378: [Core] New design for ND explore bar
**Is your feature request related to a problem? Please describe.**
The current "Explore bar" for the North Dakota dashboard takes up a fair amount of vertical space and lacks labels for the selections. This has worked for users so far, but it's not an ideal experience. 

**Describe the solution you'd like**
Proposing a new look for the filter bar similar to the Lantern filter bar. Mock attached below. 

**_The filter options left to right should be:_**

- Count vs rate
- Time
- Supervision type
- Office

Note: **in the mock, the time period and supervision type selections have a switched order from what I want.** The mock also has a different navigation structure than the live dashboard, so below is a modified mock with the filter bar in the live version, but with too much gray space between the filter bar and the important notes bar

As the page is narrowed, the size of the text of the filter titles should get smaller to take up less space, similar to the Lantern bar behavior.

### PR DESCRIPTION
**Pull request by DmitryJakubovsky**
## Description of the change

* Added new design
* Refactored ToggleBar

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

Closes #378

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

**Comment by kbinney** [comment]
Nice! Thanks Dmitry.

A couple of tweaks (sorry this wasn't in a mock you could access info from, hopefully this helps):

1. The time period filter should be second from the left on the bar. So the first filter selection after the Count/rate grouping
2. Echoing Josh, for community-> explore, the location filter should be titled "Office" (for the options including Bismarck, Mandan, etc). For facilities -> explore, the location filter should be titled "County of Residence" (for the options including Adams county, barnes county, etc)
3. The filter bar should not have any space between the left side bar and it, nor between the header bar and it. The vertical space between the filter bar and the important notes bar should be the space as the vertical space between the important notes bar and the graphs. 
3. I have some tweaks to font colors, sizes, etc. I grabbed the CSS attributes for each component from the design file (unfortunately not figma), so here goes: 

- Count & rate words: 
opacity: 0.8;
font-family: Roboto-Medium;
font-size: 14px;
color: #7C8695;
letter-spacing: 0.79px;
line-height: 17px;

- selection titles (Time period, supervision type, office / county of residence)
font-family: Roboto-Medium;
font-size: 16px;
color: #72777A;
letter-spacing: 0.49px;
line-height: 21px;

- Selection options (both the visible option - All, etc -  and the full set of options that appears when you click in - only probation, only parole, etc) 
opacity: 0.8;
font-family: Roboto-Medium;
font-size: 14px;
color: #7C8695;
letter-spacing: 0.79px;
line-height: 17px;

This is what it should look like for Community -> explore, note it always touching the top and side and the time period selection to left of supervision type selection:
slim copy

This is what it should look like for Facilities -> explore, note the time period selection to the left of the county of residence selection and the title of that selection.
slim copy 2


No need for shrink-on-scroll behavior and the reactive behavior when the page gets narrower works for me. 

Thanks!

File: src/components/toggles/ToggleBar.js [hunk 1]
```diff
@@ -15,59 +15,153 @@
...
-              <DistrictToggle
-                districtOffices={props.districtOffices}
-                districts={props.availableDistricts}
-                onDistrictUpdate={props.setChartDistrict}
-                stateCode={props.stateCode}
-                replaceLa={props.replaceLa}
-              />
+              {setChartDistrict && (
+                <div className="toggle-filters__filter" id="districtToggle">
+                  <span className="toggle-filters__filter-title">Office</span>
```
[the remark this answers is not in the document]
Comment (DmitryJakubovsky) [reply src/components/toggles/ToggleBar.js]: Renamed titles. `All` options were changed to universal "All" like in mockups bellow. As I understood it does not have a big matter.

File: src/components/toggles/helpers.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    value: district,
+  }))
+);
+
+export const collectOptions = (districts, districtOffices) =>
+  districtOffices
+    ? collectDistrictOfficesOptions(districtOffices)
+    : collectDistrictsOptions(districts);
+
+export const appendAllOption = (options) => [defaultDistrictOption, ...options];
```
[the remark this answers is not in the document]
Comment (DmitryJakubovsky) [reply src/components/toggles/helpers.js]: Thanks for your remark!

File: public/index.html [hunk 3]
```diff
@@ -1,82 +1,81 @@
...
       You can add webfonts, meta tags, or analytics to this file.
       The build step will place the bundled scripts into the <body> tag.
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
-  </body>
-</html>
+</body>
 
+</html>
```
Comment (jessex) [inline public/index.html:81]: We need to add this newline back to the end of the file.

But also, why did we dedent everything in this file? Did a linter run over it? Just curious.

File: src/components/controls/RadioGroup.scss [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+  input {
+    position: relative;
+    padding: 0;
+    margin: 0 8px 0 0;
+    vertical-align: bottom;
+    top: -3px;
+    *overflow: hidden;
+  }
+}
```
Comment (jessex) [inline src/components/controls/RadioGroup.scss:29]: Missing a newline here.

File: src/components/controls/Select.js [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+const fontStyles = {
+  color: "rgba(114, 119, 122, 0.8)",
+  textTransform: "uppercase",
+};
+
+const defaults = {
+  isSearchable: false,
+  styles: {
+    container: (styles) => ({ ...styles, ...fontStyles }),
+    // control: (styles) => ({ ...styles, ...fontStyles }),
```
Comment (jessex) [inline src/components/controls/Select.js:31]: We should remove these comment or use it.

File: public/index.html (hunk 3 above)
Comment (DmitryJakubovsky) [reply public/index.html:81]: Sorry for that, again. My editor persistently suggests me format this file. I switched off it.

File: src/components/controls/RadioGroup.scss (hunk 4 above)
Comment (DmitryJakubovsky) [reply src/components/controls/RadioGroup.scss:29]: Done.

File: src/components/controls/Select.js (hunk 5 above)
Comment (DmitryJakubovsky) [reply src/components/controls/Select.js:31]: Removed it.

**Comment by DmitryJakubovsky** [comment]
@jessex 

### Fonts
I think fonts were changed because I added Roboto font (@kbinney asked me about it).
Here you can see that Roboto font is default font, but it was not included before.
https://github.com/Recidiviz/pulse-dashboard/blob/e41e8f4829ea7f49423048522c9a37da7bbb6fb2/src/assets/styles/spec/settings/fonts.scss#L1-L4
After I added this font it applies. Should I disable this font?

### Icons
I checked, an avatar is a dynamic image and it received by the API. `Demo Jones` avatar has dimensions 64x64. 
avatar
I think your avatar has its own dimensions.

### Geo View

My fail. Fixed.

**Comment by jessex** [comment]
You're right about the avatar. Thanks for solving that, @DmitryJakubovsky! And thanks for fixing the map/graph button logic. 

@kbinney, can you weigh in on the fonts here? Essentially, we had a misconfiguration that Dmitry has "solved" but the solution means changing most of the fonts in the app to Roboto. Would you prefer to maintain the fonts we already have? That's the last question here before we can launch this.

**Comment by kbinney** [comment]
I would prefer to maintain the fonts we already have, but am confused if that means the explore bar will look like it did prior to iteration. Is it possible to have the explore bar look how it looks in most recent commit, and the rest of the app maintain the fonts and styles they had prior to this change?

**Comment by jessex** [comment]
>Is it possible to have the explore bar look how it looks in most recent commit
Yeah that shouldn't be a problem, it just means the Explore bar and the rest of the app will be using different fonts.

@DmitryJakubovsky once we've made that update, to revert the rest of the app to the fonts they were using previously and keep the Explore bar as is, we'll be ready to merge this.

**Pull request merged**